docs/content/preview/drivers-orms/include-drivers-orms-list.md

```diff
@@ -0,0 +1,127 @@
+| PostgreSQL JDBC Driver  | [42.3.4](https://mvnrepository.com/artifact/org.postgresql/postgresql/42.3.4) | Full | [CRUD](/preview/drivers-orms/java/postgres-jdbc/) |
+| Vert.x Pg Client       | [4.3.2](https://mvnrepository.com/artifact/io.vertx/vertx-core/4.3.2) | Full | [CRUD](/preview/drivers-orms/java/ysql-vertx-pg-client/) |
+| YugabyteDB Java Driver for YCQL | 3.10 | Full | [CRUD](/preview/drivers-orms/java/ycql) |
+| YugabyteDB Java Driver for YCQL | 4.15 | Full | [CRUD](/preview/drivers-orms/java/ycql-4.x) |
+| Ebean                   | | Full | [CRUD](/preview/drivers-orms/java/ebean/) |
+| Hibernate               | | Full | [CRUD](/preview/drivers-orms/java/hibernate/) |
+| Spring Data YugabyteDB  | | Full | [CRUD](/preview/integrations/spring-framework/sdyb/#examples) |
+| Spring Data JPA         | | Full | [CRUD](/preview/integrations/spring-framework/sd-jpa/#fundamentals) |
+| MyBatis                 | | Full | [CRUD](/preview/drivers-orms/java/mybatis/) |
+<!-- | Micronaut | Beta |  | -->
+<!-- | Quarkus | Beta |  | -->
+
+  {{% /tab %}}
+
+  {{% tab header="Go" lang="go" %}}
+
+| Driver/ORM | Version | Support Level | Example apps |
+| :--------- | :------ | :------------ | :----------- |
+| YugabyteDB PGX Smart Driver [Recommended] | [v4](https://pkg.go.dev/github.com/yugabyte/pgx/) | Full | [CRUD](/preview/drivers-orms/go/yb-pgx/) |
+| PGX Driver | [v4](https://pkg.go.dev/github.com/jackc/pgx/) | Full | [CRUD](/preview/drivers-orms/go/pgx/) |
+| PQ Driver  | [v1.10.2](https://github.com/lib/pq/releases/tag/v1.10.2/) | Full | [CRUD](/preview/drivers-orms/go/pq/) |
+| YugabyteDB Go Driver for YCQL | 3.16.3 | Full | [CRUD](/preview/drivers-orms/go/ycql) |
+| GORM       | | Full | [CRUD](/preview/drivers-orms/go/gorm/) |
+| PG         | | Full | [CRUD](/preview/drivers-orms/go/pg/) |
+
+  {{% /tab %}}
+
+  {{% tab header="Python" lang="python" %}}
+
+| Driver/ORM | Version | Support Level | Example apps |
+| :--------- | :------ | :------------ | :----------- |
+| YugabyteDB Psycopg2 Smart Driver [Recommended] | 2.9.3 |Full | [CRUD](/preview/drivers-orms/python/yugabyte-psycopg2/) |
+| PostgreSQL Psycopg2 Driver | 2.9.3 | Full | [CRUD](/preview/drivers-orms/python/postgres-psycopg2/) |
+| aiopg      | | Full | [Hello World](/preview/drivers-orms/python/aiopg/) |
+| Django     | | Full | [CRUD](/preview/drivers-orms/python/django/) |
+| SQLAlchemy | | Full | [CRUD](/preview/drivers-orms/python/sqlalchemy/) |
+
+  {{% /tab %}}
+
+  {{% tab header="NodeJS" lang="nodejs" %}}
+
+| Driver/ORM | Version | Support Level | Example apps |
+| :--------- | :------ | :------------ | :----------- |
+| YugabyteDB node-postgres Smart Driver [Recommended] | [8.7.3-yb-1](https://www.npmjs.com/package/pg) | Full | [CRUD](/preview/drivers-orms/nodejs/yugabyte-node-driver/) |
```

Comment:
[8.7.3-yb-1](https://www.npmjs.com/package/pg) - Doesn't look like the right link.